### PR TITLE
APA: Add "personal communication" localization

### PR DESCRIPTION
**Pull request by bwiernik**
cf. https://forums.zotero.org/discussion/69430/citing-interviews

### CODE REVIEW

**Comment by bwiernik** [comment]
Happy to add other locales if folks can share the correct phrasing.

File: apa-annotated-bibliography.csl [hunk 1]
```diff
@@ -51,11 +51,21 @@
...
     <terms>
       <term name="et-al">et al.</term>
+      <term name="letter">persönliche Kommunikation</term>
+      <term name="letter" form="short">Brief</term>
     </terms>
   </locale>
   <locale xml:lang="es">
     <terms>
       <term name="from">de</term>
+      <term name="letter">persönliche Kommunikation</term>
```
Comment (adam3smith) [inline apa-annotated-bibliography.csl]: that's not right ;)

Comment (bwiernik) [reply apa-annotated-bibliography.csl]: lol. Oops. My copy paste skills are amazing.

File: apa-annotated-bibliography.csl [hunk 2]
```diff
@@ -51,11 +51,21 @@
...
+      <term name="letter" form="short">carta</term>
+    </terms>
+  </locale>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="letter">comunicación personal</term>
```
Comment (adam3smith) [inline apa-annotated-bibliography.csl]: (nor is this; I think that should be "communication personnelle" but as I've mentioned, my French is pretty basic.

Comment (bwiernik) [reply apa-annotated-bibliography.csl]: Yeah, copy-pasted into the french field instead of spanish..

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**